app/assets/json/translations/react/es.json

```diff
@@ -946,7 +946,7 @@
   "listings.includesPriorityUnits": "Incluye unidades con prioridad para %{priorities}",
   "listings.includingChildren": "(incluidos <span class='eligibility-subtext'>%{number} %{children}</span> menores de 6 años)",
   "listings.incomeExceptions.intro": "Las personas en su hogar pueden necesitar <a href='%{url}' target='_blank'> cálculos de ingresos especiales</a> en el caso que ellos",
-  "listings.incomeExceptions.nontaxable": "Reciban <span data-tooltip='%{tooltip}' title='%{tooltip}' aria-haspopup='true' class='has-tip'>ingresos no reportables</span>",
+  "listings.incomeExceptions.nontaxable": "Reciban ingresos no reportables",
```

Comment:
The version of this text that had a span was removed 13 months ago: https://github.com/SFDigitalServices/sf-dahlia-web/commit/309ad3b1401d590ffdcec2a182b224a6f2bf57a6#diff-aef4757754c858d1a9ffa5d6e0f6bbb4b9dbe9791e92a7e41b65baa053707e9dR824

I'm wondering if there is some oversight in our translation process that is causing these issues.